googlemock/docs/cheat_sheet.md

```diff
@@ -490,12 +490,12 @@ which must be a permanent callback.
 #### Returning a Value
 
 <!-- mdformat off(no multiline tables) -->
-|                             |                                               |
+| Action                      | Description                                   |
 | :-------------------------- | :-------------------------------------------- |
 | `Return()`                  | Return from a `void` mock function.           |
-| `Return(value)`             | Return `value`. If the type of `value` is     different to the mock function's return type, `value` is converted to the latter type <i>at the time the expectation is set</i>, not when the action is executed. |
+| `Return(value)`             | Return `value`. If the type of `value` is different to the mock function's return type, `value` is converted to the latter type *at the time the expectation is set*, not when the action is executed. |
```

Comment:
I can see you tried to keep the ending `|` aligned to adjacent ones, but here (and in some other places too) there is a small inconsistency.

On the other hand, this line didn't fit because of how long the description is. Maybe keeping everything aligned would make all other short descriptions unreadable or unnecessarily long.

What do you think? Do you have any other ideas?

Reply:
Thank you for your feedback!

I tried to align all the tables in all the documents once!
But as you know, some of them were still impossible and even they were less readable
For that reason, I gave up and just aligned a few tables which have other modifications as well
It is also possible to put them back if you give me further feedback!

And I think using html <table> might be an option to describe table neatly cuz github does not support multiline table!

Reply:
And so I thought ;) Btw. I like your idea with http tables.